pytube/exceptions.py

```diff
@@ -80,5 +80,16 @@ def __init__(self, video_id: str):
         self.video_id = video_id
 
 
+class MembersOnly(PytubeError):
+    """Video is members-only."""
```

Comment:
nit: clarity.

Video is members-only -> Video is members-only,  a special type of video only viewable to users who have subscribed to a content creator. https://support.google.com/youtube/answer/7544492?hl=en

Reply:
Yeah, I'll make this more descriptive.